site/data/contributors.yml

```diff
@@ -0,0 +1,42 @@
+- id: 20046
+  login: fahad19
+  html_url: 'https://github.com/fahad19'
+  contributions: 77
+  name: Fahad Ibnay Heylaal
+  avatar_url: 'https://avatars1.githubusercontent.com/u/20046?v=3'
+- id: 1002056
+  login: mAiNiNfEcTiOn
+  html_url: 'https://github.com/mAiNiNfEcTiOn'
+  contributions: 15
+  name: Ricardo Machado
+  avatar_url: 'https://avatars2.githubusercontent.com/u/1002056?v=3'
+- id: 1031418
+  login: alexmiranda
+  html_url: 'https://github.com/alexmiranda'
+  contributions: 11
+  name: Alex Miranda
+  avatar_url: 'https://avatars0.githubusercontent.com/u/1031418?v=3'
+- id: 5013232
+  login: heidilaw4
+  html_url: 'https://github.com/heidilaw4'
+  contributions: 1
+  name: Heidi Law
+  avatar_url: 'https://avatars1.githubusercontent.com/u/5013232?v=3'
+- id: 1122274
+  login: markvincze
+  html_url: 'https://github.com/markvincze'
+  contributions: 1
+  name: Mark Vincze
+  avatar_url: 'https://avatars0.githubusercontent.com/u/1122274?v=3'
+- id: 813786
+  login: paulcodiny
+  html_url: 'https://github.com/paulcodiny'
+  contributions: 1
+  name: Pavlik Kiselev
```

Comment:
Shouldn't we ask Pavlik at first? I don't know if he would agreed about being on the Frint website